src/wooting-usb.c

```diff
@@ -168,6 +205,12 @@ bool wooting_usb_send_feature(uint8_t commandId, uint8_t parameter0, uint8_t par
 	if (!wooting_usb_find_keyboard()) {
 		return false;
 	}
+	// prevent sending unnecessary data to the wootings if the index exceedes it's capabilities
```

Comment:
hm. get_safe_led_idex already ensures this does not happen.
it would just result in code that would rarely need to filter.
just tell me how you think about this. in a personal project i'd probably not do this. but since this is a public sdk i thought it is reasonable.